test/e2e/.mocharc_playwright.yml

```diff
@@ -0,0 +1,9 @@
+require:
+  - 'esm'
+  - 'test/mocha.env.js'
+  - 'mocha-steps'
+file:
+  - 'lib/before.js'
+  - 'lib/hooks-playwright/hooks.ts'
+reporter:
+  - 'mocha-teamcity-reporter'
```

Comment:
Note that some of the command line arguments under `mocha_args` in TeamCity has been moved here.

Reply:
The goal of having some args here and some pass in from TeamCity's shell command is that the arguments here should be the defaults when a developer runs the tests in their machine, and have the TeamCity specific things (like the reporter) only in the shell command.